docs/index.rst

```diff
@@ -11,7 +11,7 @@ Contents:
    celery
    wsgi
    email
-   search
+   elasticsearch
```

Comment:
Renaming that since "search" is a protected name for Sphinx: http://sphinx-doc.org/markup/toctree.html#special-names